### DISCUSSION TITLE
Copy enrolled devices from one network to another?

### DISCUSSION THREAD

**Discussion by ripvega** [ESPresense/ESPresense#1241] [Support]
I am setting up ESPresence on another network.

Is it possible to copy the enrolled devices from my original network and publish them to a new network so that I don't have to go through the enrolling process for each device again?

If so, what is the recommended way to accomplish this?

**Comment by Cyberbeni** [comment]
Open MQTT explorer, copy the topics from `espresense/settings/irk:xxx/config` and their contents then publish the same topics with the same contents on the other mqtt network. (Make sure that you check "retain" when publishing)
